### PR TITLE
Development

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+  width: 100%;
+  vertical-align: middle;
+  background: #b8b8b8;
+}
+
+p {
+  font-family: "Indie Flower", cursive;
+  font-size: 26px;
+  line-height: calc(18px * 1.5);
+  font-weight: 400;
```
Comment (mcrd25) [inline assets/css/style.css]: For lines 22, 23, 25 the shorthand font could be used instead and save lines of code 😄 . If of course, you prefer it like this sure, but may be useful to use `font` instead,.

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+  text-align: justify;
+  text-indent: 2em;
+}
+
+.f-s {
+  font : 18px/1.5 "Lora", serif;
```
Comment (mcrd25) [inline assets/css/style.css]: Nice ✅ 👍

File: assets/css/style.css [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+  text-align: justify;
+  text-indent: 3em;
+}
+
+.f-m {
+  font-family: "Lora", serif;
```
Comment (mcrd25) [inline assets/css/style.css]: `font` can and maybe should be used instead of `font-family` and `font-size`

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Project Well-done 👍 

-Address comment review (free to disagree)
- The project is well, but I think maybe you should consider Don’t Miss These Latest
Articles section seeing as it is one of the first things one sees on the website. 
- I realize you only have used black for text, but consider using the differing font-weight base on the importance
- I noticed there is no section that is black to you, is it that none of the sections caught your attention first?
- don't forget white is in greyscale as well
- For me, there is not much contrast in greys you have used

Really great first attempt. 👍 You can almost 🚀  but please address review comments and submit another review request 😄 .

## If you disagree please I am open to it, comment here or message me on slack @maya.

[Here](https://mcrd25.github.io/smashing-heat-map/) is what ours looked like, not that yours should look like this. This was confusing for us at first but we later did more reading on UIUX to help us really understand why we were doing what we were doing. This is really based on what you guys think is the most important elements on the page.

**Comment by ghost** [comment]
We changed "font" part and "Weight more at important" parts. Please review again our code. Thank you.

**Review by mcrd25 — APPROVED** [review]
# You guys may 🚀 . Great job! 👍 

- I really liked that you guys made it responsive as well. 👍 
- README is well done
- Implemented changes
- Overall, well done

You may 🚢 it and move on. 🔝 👍 🚀 🥇

**Pull request merged**